docs/cli.md

```diff
@@ -275,6 +275,57 @@ docker run --rm -ti \
 - **File Downloads**: Agent-collected files
 - **Screenshots**: Visual captures from agents
 
+## Cobalt Strike Connector
+
+Ingest data from Cobalt Strike into Nemesis.
```

Comment:
Mind adding some info here stating it requires the REST API and then linking to how to set that up?